internal/validate/string.go

```diff
@@ -24,6 +24,16 @@ type StringRule struct {
 	CharacterRanges []CharRange
 }
 
+func String(value, name string, min, max int, charset ...CharRange) StringRule {
```

Comment:
This seems harder to use than the struct literal. Functions with more than 3 arguments generally are. I'd suggest using the struct literal directly.

I'm not seeing this used in this PR.

Reply:
I ended up writing it for something I delayed. I don't find the struct literal very clean looking in use.